.github/workflows/build.yml

```
@@ -0,0 +1,41 @@
+name: Build
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+
+    - uses: actions/checkout@v2
+      with:
+        repository: clugg/sm-json
+        path: 'json'
+
+    - uses: actions/checkout@v2
+      with:
+        path: 'get5'
+
+    - uses: rumblefrog/setup-sp@master
+      with:
+        version: '1.10.x'
+
+    - run: wget https://raw.githubusercontent.com/KyleSanderson/SteamWorks/master/Pawn/includes/SteamWorks.inc -P steamworks
```

Pin the version used to avoid external changes breaking the building
```suggestion
    - uses: actions/checkout@v2
      with:
        repository: clugg/sm-json
        path: 'json'
        ref: 3a9e22b8b4056809c95b3329e9f779b9c0be34cc

    - uses: actions/checkout@v2
      with:
        path: 'get5'

    - uses: rumblefrog/setup-sp@50d937a464d62aed807037f61c1a5c9d59964f88
      with:
        version: '1.10.x'

    - run: wget https://raw.githubusercontent.com/KyleSanderson/SteamWorks/9fb35e2f96e1e539cb1c9c769b5dc5edf5d50b4b/Pawn/includes/SteamWorks.inc -P steamworks
```

---

1. pinned steamworks to 1.2.3c tag
2. sm-json now loaded from submodule